aif360/sklearn/explainers/bias_explainer.py

```diff
@@ -0,0 +1,193 @@
+from fairxplainer.fair_explainer import FairXplainer
+from fairxplainer.fair_explainer import plot as fif_plot
+import numpy as np
+
+
+def explain_statistical_parity(
+        clf, 
+        X, 
+        sensitive_features, 
+        maxorder=2, 
+        spline_intervals=5,
+        k=7,
+        seed=22, 
+        cpu_time=300, 
+        verbose=False):
+    """
+        Explain the statistical parity of a classifier given a dataset
+
+    Args:
+        clf: the trained model to explain
+            clf.predict() should be implemented
```

Comment:
does it matter if this doesn't return 0/1? what if 0 is favorable?

Reply:
The code is implemented in a way that 1 is the favourable class. Also, I assume discrete (particularly Boolean) prediction by the classifier.